frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -307,11 +267,38 @@ const ManagePolicyPage = ({
       enabled: isRouteOk && isPremiumTier && !!teamIdForApi,
       onSuccess: (data) => {
         setTeamPolicies(data.policies);
-        setInheritedPolicies(data.inherited_policies);
       },
     }
   );
 
+  const {
+    data: teamPoliciesCountIncludeInherited,
+    isFetching: isFetchingTeamCountIncludeInherited,
+    refetch: refetchTeamPoliciesCountIncludeInherited,
+  } = useQuery<
+    IPoliciesCountResponse,
+    Error,
+    number,
+    ITeamPoliciesCountQueryKey[]
+  >(
+    [
+      {
+        scope: "teamPoliciesCountIncludeInherited",
+        query: searchQuery,
+        teamId: teamIdForApi || 0, // TODO: Fix number/undefined type
+        includeInherited: !!teamIdForApi,
+      },
+    ],
+    ({ queryKey }) => teamPoliciesAPI.getCount(queryKey[0]),
```

Comment:
> Not getting an empty state when no policies are present, though this could be a figment of how I'm manipulating the test data. Worth double-checking the empty states still

@jacobshandling samesies for empty state, once the API is implemented, empty state should show correctly from the count coming from the API